### BIG differences, this is diff of wrong files.
### CLUS "Condition Type ERROR"
### and CLHS topic selection, I don't find it
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:error.txt	2017-04-27 21:59:54.000000000 +0100
+++ saved-pages-clus/cl:types:error.txt	2017-04-27 21:59:54.000000000 +0100
@@ -1,4 +1,13 @@
-ERROR
-Please select which reference to ERROR you intended
+Condition Type ERROR

-Condition TypeFunctionGlossary Entry
\ No newline at end of file
+Class Precedence List
+error, serious-condition, condition, t
+Description
+The type error consists of all conditions that represent errors.
\ No newline at end of file
